### PR TITLE
[OCPBUGS30325] Assisted Installer - add missing supported operators

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s): 4.14+

Issue:
- https://issues.redhat.com/browse/OCPBUGS-30325

Link to docs preview:

https://73468--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_on_prem_assisted/installing-on-prem-assisted

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/assisted-installer-using-the-assisted-installer.adoc [hunk 1]
```diff
@@ -37,7 +37,13 @@ The {ai-full} supports installing {product-title} on premises in a connected env
 * Highly available {product-title} or {sno} (SNO)
 
 * {product-title} on bare metal, Nutanix, or vSphere with full platform integration, or other virtualization platforms without integration
-* Optionally {VirtProductName} and {rh-storage} (formerly OpenShift Container Storage)
+
+* Optionally, {VirtProductName}, {rh-storage}, multicluster engine, and {lvms-first}
++
+[NOTE]
+====
+Currently, {VirtProductName} and {lvms} are not supported on {ibm-z-name} (`s390x`) architecture.
```
Comment (AmadeusPodvratnik) [inline modules/assisted-installer-using-the-assisted-installer.adoc:45]: The order on the panel is: OpenShift Virtualization, multicluster engine, Logical Volume Manager (LVM) Storage and OpenShift Data Foundation. Not sure if this matters but I thought worth to be mentioned.

**Review by AmadeusPodvratnik — APPROVED** [review]

File: modules/assisted-installer-using-the-assisted-installer.adoc [hunk 2]
```diff
@@ -37,7 +37,13 @@ The {ai-full} supports installing {product-title} on premises in a connected env
 * Highly available {product-title} or {sno} (SNO)
 
 * {product-title} on bare metal, Nutanix, or vSphere with full platform integration, or other virtualization platforms without integration
-* Optionally {VirtProductName} and {rh-storage} (formerly OpenShift Container Storage)
+
+* Optionally, {VirtProductName}, multicluster engine, {lvms-first}, and {rh-storage}
```
Comment (bburt-rh) [inline modules/assisted-installer-using-the-assisted-installer.adoc:41]:
```suggestion
* Optional: {VirtProductName}, multicluster engine, {lvms-first}, and {rh-storage}
```
I think this is the standard way we refer to optional items in a numbered or bulleted list.

**Review by bburt-rh — APPROVED** [review]
LGTM with one nitpick. 

Also, please don't forget to add the OCP docs versions that this change applies to. Thank you!

**Comment by michaelryanpeter** [comment]
LGTM, but we need to know which OCP versions this applies to. Please update comment zero with this information. Feel free to DM me in Slack after you have updated that information. Thanks!

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Comment by SNiemann15** [comment]
> LGTM, but we need to know which OCP versions this applies to. Please update comment zero with this information. Feel free to DM me in Slack after you have updated that information. Thanks!
> 
> /remove-label merge-review-in-progress /remove-label merge-review-needed

@AmadeusPodvratnik To which versions should this be backported?

**Comment by AmadeusPodvratnik** [comment]
> > LGTM, but we need to know which OCP versions this applies to. Please update comment zero with this information. Feel free to DM me in Slack after you have updated that information. Thanks!
> > /remove-label merge-review-in-progress /remove-label merge-review-needed
> 
> @AmadeusPodvratnik To which versions should this be backported?

I would say to 4.14.

**Comment by michaelryanpeter** [comment]
NOTE: @SNiemann15 asked for a merge in a private message in Slack
/label merge-review-in-progress

**Pull request merged**